bin/get_workspace_status

```diff
@@ -51,14 +51,10 @@ if [[ -n ${ISTIO_DOCKER_HUB} ]]; then
   DOCKER_HUB="${ISTIO_DOCKER_HUB}"
 fi
 
-# used by pilot/istioctl/security version package
-echo buildAppVersion    "${VERSION}"
+# used by pkg/version
+echo buildVersion       "${VERSION}"
```

Comment:
If you really want to change this name then I think you'll also need to make a similar name change for buildAppVersion in pkg/version/version.go